WEAPON/CreateWeaponObject.md

```diff
@@ -8,23 +8,26 @@ ns: WEAPON
 Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, Any p7);
 ```
 
+Creates a weapon object in the world at the specified location.
+
 ```
 Now has 8 params.  
 ```
 
+**Note** To create a networked weapon object and attach it to the player, use [CREATE_OBJECT](#_0x509D5878EB39E842) instead.
+For more details see [Weapon objects poor sync](https://github.com/citizenfx/fivem/issues/2249).
+
 ```
 NativeDB Added Parameter 9: Any p8
 NativeDB Added Parameter 10: Any p9
 ```
 
 ## Parameters
-* **weaponHash**: 
-* **ammoCount**: 
-* **x**: 
-* **y**: 
-* **z**: 
-* **showWorldModel**: 
-* **scale**: 
-* **p7**: 
-
-## Return value
+* **weaponHash**: The hash of the weapon to create.
+* **ammoCount**: The amount of ammo for the weapon.
+* **x**: X coordinate for the weapon's position.
+* **y**: Y coordinate for the weapon's position.
+* **z**: Z coordinate for the weapon's position.
+* **showWorldModel**:
+* **scale**:
+* **p7**:
```

Comment:
After some research, I have seen that `p7` is the script host. (R* always set it to `true`)